src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -293,6 +282,54 @@ internal static string FindBestVersionMatch(VersionRange versionRange, IEnumerab
             return matchingVersion?.ToString();
         }
 
+        private NuGetVersion ResolvePlatformReleaseChannelVersion(VersionRange versionRange, IEnumerable<string> versions)
+        {
+            var bundleVersions = versions.Select(p =>
+            {
+                var dirName = Path.GetFileName(p);
+                NuGetVersion.TryParse(dirName, out NuGetVersion version);
+                if (version != null)
+                {
+                    version = versionRange.Satisfies(version) ? version : null;
+                }
+                return version;
+            }).Where(v => v != null).OrderByDescending(version => version.Version).ToList();
+
+            return _platformReleaseChannel.ToUpperInvariant() switch
+            {
+                ScriptConstants.StandardPlatformChannelNameUpper or ScriptConstants.ExtendedPlatformChannelNameUpper => GetStandardOrExtendedBundleVersion(bundleVersions),
+                ScriptConstants.LatestPlatformChannelNameUpper or "" => GetLatestBundleVersion(bundleVersions),
+                _ => HandleUnknownPlatformReleaseChannelName(bundleVersions)
+            };
+        }
+
+        private NuGetVersion GetStandardOrExtendedBundleVersion(IList<NuGetVersion> orderedByDescBundlesList)
+        {
+            if (orderedByDescBundlesList.Count() > 1)
```

Comment:
Use the `Count` property since you are dealing with the IList now (The `Count()` method might be internally returning the `Count` in the implementation, but it's not best to make assumptions about the implementation).